examples/consumer.py

```diff
@@ -18,24 +18,55 @@
 #
 # Example high-level Kafka 0.9 balanced Consumer
 #
-
 from confluent_kafka import Consumer, KafkaException, KafkaError
 import sys
+import getopt
+import json
+from pprint import pformat
 
-if __name__ == '__main__':
-    if len(sys.argv) < 4:
-        sys.stderr.write('Usage: %s <bootstrap-brokers> <group> <topic1> <topic2> ..\n' % sys.argv[0])
-        sys.exit(1)
+def stats_cb(stats_json_str):
+    stats_json = json.loads(stats_json_str)
+    print('\nKAFKA Stats: {}\n'.format(pformat(stats_json)))
 
-    broker = sys.argv[1]
-    group  = sys.argv[2]
-    topics = sys.argv[3:]
+def print_usage_and_exit(program_name):
+    sys.stderr.write('Usage: %s [options..] <bootstrap-brokers> <group> <topic1> <topic2> ..\n' % program_name)
+    options='''
+ Options:
+  -T <intvl>   Enable statistics from Kafka at specified interval (ms)   
```

Comment:
the stats are from the client, not Kafka, so might want to rephrase that to something like:
`Enable client statistics at specified interval (ms)`